components/gitpod-db/go/oidc_client_config.go

```diff
@@ -102,6 +102,7 @@ func ListOIDCClientConfigsForOrganization(ctx context.Context, conn *gorm.DB, or
 		WithContext(ctx).
 		Where("organizationId = ?", organizationID).
 		Where("deleted = ?", 0).
+		Order("id").
```

Comment:
This is done for now, to ensure the output is stable. We'll likely want to sort by CreationTime or something like that, which we don't yet have as a column.